Allwmake

```diff
@@ -48,38 +35,44 @@ log "Building the OpenFOAM adapter for preCICE..."
 # 5. translate the new line characters to "\", i.e. make everything again one line.
 
 # Extract the paths from the CPLUS_INCLUDE_PATH list and convert them to a sequence of "-I/path/ \"
-if [ "${CPLUS_INCLUDE_PATH}" ]; then
-    ADAPTER_GLOBAL_CPLUS_INC_PATHS=$(echo $CPLUS_INCLUDE_PATH | tr ':' '\n' |  sed '/^$/d' | sed 's/^/-I/' | tr '\n' ' \\')
+if [ "${CPLUS_INCLUDE_PATH:-}" ]; then
+    ADAPTER_GLOBAL_CPLUS_INC_PATHS=$(echo "${CPLUS_INCLUDE_PATH}" | tr ':' '\n' | sed '/^$/d' | sed 's/^/-I/' | tr '\n' ' \\ ')
+else
```

Comment:
```suggestion
else if
elif [ "${CPATH:-}" ]; then
    ADAPTER_GLOBAL_CPLUS_INC_PATHS=$(echo "${CPATH}" | tr ':' '\n' | sed '/^$/d' | sed 's/^/-I/' | tr '\n' ' \\ ')
else
```

Reply:
Is there a good practice reason behind this suggestion? Any citations? See my comment below, I would rather remove this altogether, if possible.

Reply:
`CPATH` is for `C` and `c++`

https://stackoverflow.com/questions/558803/how-to-add-a-default-include-path-for-gcc-in-linux

Reply:
We have an issue here: apparently, `CPLUS_INCLUDE_PATH` does not have any effect, but `CPATH` does. I am investigating this.

Reply:
I thought you want to skip the `CPATH`?